tests/countries/test_anguilla.py

```
@@ -0,0 +1,330 @@
+            "2018-05-30",
+            "2019-05-30",
+            "2020-06-02",
+            "2021-05-31",
+            "2022-05-30",
+            "2023-05-30",
+            "2024-05-30",
+            "2025-05-30",
+        )
+        self.assertHolidayName(name, range(2010, 2050))
+
+    def test_sovereignty_day(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)